### ISSUE TITLE
Fire Emblem Warriors: Three Hopes - Save/Load not respond

### ISSUE THREAD

**Issue by M4rQu1Nh0S** [yuzu-emu/yuzu#10033]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Affected Build(s)

Mainline 1395

### Description of Issue

After choosing to save or load any save data, the game not process the request, so the game keeps running and not respond to any command, the UI hides.

### Expected Behavior

After select save, the game have to write the save file to player keeps playing or quit
After select Load, the game have to read the save file and load the game.

### Reproduction Steps

Basically select load a saved data at main menu or save while the character are in the camp.

### Log File

[yuzu_log.txt](https://github.com/yuzu-emu/yuzu/files/11182176/yuzu_log.txt)

### System Configuration

CPU: Intel i7 2600
GPU/Driver: Nvidia 1050ti 4gb / Driver 525.89.02
RAM: 16GB
OS: Linux Debian Bookworm.

**Comment by gidoly** [comment]
this is the bug from 1.0.0 version of game you have to install lastest update of game

**Comment by M4rQu1Nh0S** [comment]
Fixed with update 1.0.1.
